packages/fxa-oauth-console/package.json

```diff
@@ -63,6 +63,7 @@
     "grunt-copyright": "0.3.0",
     "grunt-eslint": "19.0.0",
     "load-grunt-tasks": "3.5.2",
+    "natives": "1.1.6",
```

Comment:
Did you want to commit the `package-lock.json` too? Testing this locally, my `package-lock.json` natives version got updated from 1.1.4

Reply:
Ah, sure, I could do that.